hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java

```diff
@@ -126,6 +127,17 @@
             ImmutableSet.of("newSecurityException")
     );
 
+    private static final Set<String> ignoreCheck = new CopyOnWriteArraySet<>();
+
+    public static void addIgnoreCheck(String clazz) {
+        if (callFromGremlin()) {
+            throw newSecurityException(
+                  "Not allowed to add ignore check via Gremlin");
```

Comment:
add ignore-checked class

Reply:
I think this method be called when inited best, if add ignore-checked, maybe it is called on other time;